apps/src/pythonlab/layout/layout.module.scss

```diff
@@ -0,0 +1,20 @@
+.layoutContainer {
+  display: flex;
+  flex-direction: row;
+  height: 100%;
+  width: 100%;
+}
+
+.flexColumn {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+}
+
+.flexGrow {
+  flex-grow: 1;
+}
+
+.flexShrink0 {
```

Comment:
I'm trying to understand behavior I'm observing in `VerticalLayout`. 
When the output panel is at max width, I cannot drag the left resize bar. However, when the info panel is at max width, I can adjust the right bar. See screencast vid below. I see that both the info and output panel have the same style rule applied. Do you mind explaining this behavior?

https://github.com/user-attachments/assets/02bce9c5-89cf-4647-b8b0-7036ba7045ee

Reply:
Yeah it has to do with the math of the panels, I debated the best approach here. Basically the 3 panels have priority in terms of sizing: 1st is editor, second is console, third is instructions. First the editor looks at how much space it has and either makes itself width of the available space or 400px, whichever is bigger. Then the console sees how much space it has, accounting only for the minimum info panel size, but the adjusted editor size. Then the info panel takes up the remaining space. See the `useEffect` in `VerticalLayout` for the code.

Reply:
That makes sense - thanks!